### PR TITLE
CSW snark proof inputs list changed.

### PR DESCRIPTION
**Pull request by i-Alex**

### CODE REVIEW

**Review by alsala — APPROVED** [review]

File: src/gtest/test_validation.cpp [hunk 1]
```diff
@@ -53,8 +53,7 @@ class FakeCoinsViewDB : public CCoinsView {
                     CNullifiersMap &mapNullifiers,
                     CSidechainsMap& mapSidechains,
                     CSidechainEventsMap& mapSidechainEvents,
-                    CCswNullifiersMap& cswNullifiersss,
-                    CCertDataHashMap& certDataHashes) override
+                    CCswNullifiersMap& cswNullifiersss) override
```
Comment (abi87) [inline src/gtest/test_validation.cpp:56]: Why all these ssss? XD

**Review by abi87 — APPROVED** [review]

**Comment by abi87** [comment]
I see two UTs failing: 
[  FAILED  ] SidechainTestSuite.CSWTxInvalidEpoch
[  FAILED  ] SidechainTestSuite.CertificateUpdatesCertHashData
Is it foreseen?

**Comment by i-Alex** [comment]
> I see two UTs failing:
> [ FAILED ] SidechainTestSuite.CSWTxInvalidEpoch
> [ FAILED ] SidechainTestSuite.CertificateUpdatesCertHashData
> Is it foreseen?

Yes. I expect them to work after the MBTRs merge.

**Comment by abi87** [comment]
> 
> 
> > I see two UTs failing:
> > [ FAILED ] SidechainTestSuite.CSWTxInvalidEpoch
> > [ FAILED ] SidechainTestSuite.CertificateUpdatesCertHashData
> > Is it foreseen?
> 
> Yes. I expect them to work after the MBTRs merge.

@alsala just reminded me you mentioned this AFK. Sorry it slipped out of my mind

**Pull request merged**